docs/standalone.md

```diff
@@ -28,10 +28,45 @@ cd noobaa-core
 npm install
 npm run build
 # optional package everything into a single-executable at build/noobaa-core
-npm run pkg 
+npm run pkg
+sudo cp src/deploy/standalone/noobaa_configured_dns.conf /etc
```

Comment:
this can be removed in favor of #7305

Reply:
I guess also the empty src/deploy/standalone/noobaa_configured_dns.conf can be removed also

Reply:
Thanks. I removed it from both repository and the doc.